### PR TITLE
- added rename (rename-path).

### PR DESCRIPTION
**Pull request by drj7z**

### CODE REVIEW

File: class/TransmissionRPC.class.php [hunk 1]
```diff
@@ -316,6 +316,47 @@ public function move ( $ids, $target_location, $move_existing_data = true )
...
+   * Response arguments: "path", "name", and "id", holding the torrent ID integer
+   *
+   * @param int|array ids A 1-element list of transmission torrent ids
+   * @param string path The path to the file or folder that will be renamed
+   * @param string name The file or folder's new name
+   */
+  public function rename ( $ids, $path, $name )
+  {
+    if ( !is_array( $ids ) ) $ids = array( $ids );  // Convert $id to an array if only a single id was passed
+    if ( count($ids) !== 1 ) {
```
Comment (brycied00d) [inline class/TransmissionRPC.class.php]: The style, such as it is, is for there to be spaces around arguments passed to functions. `count( $ids )`

Comment (brycied00d) [inline class/TransmissionRPC.class.php]: The style, such as it is, is for function arguments to have spaces around them. `count( $ids )`

**Comment by drj7z** [comment]
Style fixed as requested.

**Comment by drj7z** [comment]
Style fixed as requested.

**Pull request merged**